js/interface-lists.js

```diff
@@ -208,6 +208,19 @@ function validate(value) {
   return false;
 }
 
+function showOrHideError (showError, element) {
+  if (showError) {
+    var $element = $(element);
+    $element.addClass('has-error');
+    $element.parents('.form-group').addClass('has-error');
+    $element.parents('.panel').addClass('panel-danger').removeClass('panel-default');
+    return;
+  }
+    $('.has-error').removeClass('has-error');
```

Comment:
Unindent this section, and add a linebreak after the `if` clause

Reply:
@tonytlwu 
Added a linebreak after `if` clause

Reply:
Also unindent this section, like below

```suggestion
  $('.has-error').removeClass('has-error');
```

Reply:
@tonytlwu 
Fixed